src/Main.elm

```diff
@@ -208,6 +205,7 @@ viewToast attributes toast =
                 [ Html.div
                     [ Attr.attribute "role" "alert"
                     , Attr.class "alert alert-error animate-in slide-in-from-top duration-500 animate-out slide-out-to-top mb-2.5"
+                    , Events.onClick (ToastMsg <| Toast.exit toast.id)
```

Comment:
For some reason, sometimes the toasts would get stuck in the screen, with this change at least we can close them by clicking on them! 🚀